pages/styles/work-ad.js

```diff
@@ -1,58 +1,66 @@
-import css from 'styled-jsx/css'
+import css from "styled-jsx/css";
+import { breakpoints } from "../../pages/styles/breakpoints.js";
```

Comment:
uff da! We need to consider this. Not that I am a fan of sorting files, But I am not sure that is the best path we could have.